### PR TITLE
Create Visual Studio 2022 solution and projects for each utility

### PR DESCRIPTION
**Pull request by kpieckiel**
Added cryptopp as a submodule and created a VS 2022 solution that contains the cryptopp cryptlib project and a project for each hash utility.

### CODE REVIEW

File: msvc/md2sum.vcxproj [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      <SubSystem>Console</SubSystem>
+      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <OptimizeReferences>true</OptimizeReferences>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <AdditionalLibraryDirectories>$(SolutionDir)cryptopp\$(Platform)\Output\$(Configuration)\</AdditionalLibraryDirectories>
+      <AdditionalDependencies>cryptlib.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ClCompile Include="..\..\..\Private\hashtest\src\utils.cpp" />
```
Comment (hgruniaux) [inline msvc/md2sum.vcxproj:162]: This should be changed to `<ClCompile Include="..\src\utils.cpp" />`

File: msvc/md2sum.vcxproj [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+    <ClCompile Include="..\src\md2.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\..\..\Private\hashtest\src\utils.hpp" />
```
Comment (hgruniaux) [inline msvc/md2sum.vcxproj:166]: Same here `<ClInclude Include="..\src\utils.hpp" />`

**Review by hgruniaux — CHANGES_REQUESTED** [review]
Thanks for the pull request!

Some remarks:
 - Actually the project `md2sum` do not compile. See the comments on msvc/md2sum.vcxproj
 - Maybe modify the CMakeLists.txt to use the git submodule instead of a vcpkg package for cryptopp if you want.

**Comment by kpieckiel** [comment]
I love peer review--thanks for catching that!  I had included those files from the wrong project directory.  This has been updated and all projects should compile now.

**Comment by hgruniaux** [comment]
Thanks for your time and effort !

**Pull request merged**